Yoast/wordpress-seo
# P3-63 Add error message when attempting to open the SEMrush modal with an empty keyphrase

## Context


* 

## Summary


This PR can be summarized in the following changelog entry:

* Displays an error message when attempting to open the SEMrush modal with an empty keyphrase.

## Relevant technical choices:

- adds a new `displayNoKeyphraseMessage` property to the redux state
- adjusts actions, reducers, selectors accordingly

## Test instructions

This PR can be tested by following these steps:

- this PR needs to be tested together with the related JS repo PR https://github.com/Yoast/javascript/pull/805
- switch the JS repo to the branch `P3-63-error-message-empty-keyphrase`
- of course, switch Yoast SEO to this branch `P3-63-error-message-empty-keyphrase`
- run `yarn link-monorepo` and then build the plugin
- edit a post
- test both in the metabox and in the sidebar 
- empty the Focus keyphrase field
- click "Get related keyphrases"
- observe the SEMrush modal doesn't open
- observe a message is displayed _above_ the keyphrase input field
- the message is "Please enter a focus keyphrase first to get related keyphrases"
- for now, the message is unstyled: **will create a new issue for the Frontend team**
- the keyphrase input field will need CSS adjustments as well to match the design: most notably, there should be a big "X" to indicate there's an error (note: this "X" is the one we usually use for errors related to input validation) and the background should be a light red
- check that other errors still work, for example:
  - enter two words separated by a comma in the keyphrase input
  - observe a message "Are you trying to use multiple keyphrase...." appears _below_ the input
  - enter a keyphrase longer than 191 characters 
  - observe a yellow alert appears _below_ the input


Screenshot: for now, the message is unstyled:

<img width="1281" alt="Screenshot 2020-07-27 at 10 56 56" src="https://user-images.githubusercontent.com/1682452/88526672-82b79900-cffc-11ea-95e8-07ab8a98b5aa.png">

Design:

<img width="430" alt="Screenshot 2020-07-27 at 10 43 47" src="https://user-images.githubusercontent.com/1682452/88526702-8ba86a80-cffc-11ea-884d-0b0641f857c9.png">



## UI changes

## Documentation

## Quality assurance


Fixes [P3-63]
